src/locales/en.json

```diff
@@ -45,19 +45,19 @@
     "remove.custom.project.not.in.list": "The selected project does not exist in the Liberty Dashboard, or it was not manually added to the Liberty Dashboard. Nothing to remove.",
     "attach.debugger.liverty.dev.in": "Attaching debugger to Liberty server in {0}.",
     "cannot.attach.debugger.to.undefined": "Cannot attach debugger to an undefined project.",
-    "liberty.dev.attach.debugger.failed": "The Liberty server might not be started. Start the Liberty server by using the 'Start' or 'Start...' command in the Liberty dashboard. If you already started the Liberty server, wait until you see the Liberty dev mode startup message and try to attach the debugger again.",
+    "liberty.dev.attach.debugger.failed": "The Liberty server might not be started. Start the Liberty server by using the 'Start' or 'Start...' command in the Liberty Dashboard. If you already started the Liberty server, wait until you see the Liberty dev mode startup message and try to attach the debugger again.",
```

Comment:
minor nit- I dont think we should captialize "dashboard", unless it appears in IDE as "Liberty Dashboard". IBM Style says only to capitalize trademarked product names. Im not sure but here I think "Liberty dashboard" is more like "Liberty server", which isn't capitalized. But if "Liberty Dashboard" appears in the UI, then it should be capitalized here.

Reply:
<img width="508" alt="Screenshot 2025-01-16 at 2 53 06 PM" src="https://github.com/user-attachments/assets/dbf7ee69-c063-4fd6-a803-6602c4d5392c" />

Reply:
Let's go with "Liberty Dashboard" as you have it